en-US/Language.xml

```diff
@@ -142,6 +161,16 @@
 
 				</listitem>
 
+			</varlistentry>
+			 <varlistentry>
+				<term>boil the ocean</term>
+				 <listitem>
+					<para>
+						Do not use. State exactly what you mean, such as "increase the scope hugely".
+					</para>
```

Comment:
tbh I would never write that and I don't think we should offer it as an example. I'm also not convinced that it's a substitute for "boil the ocean". Maybe @sffrench has some ideas?

Reply:
Removed this entry; I don't think it is needed.